README.md

```diff
@@ -136,42 +136,41 @@ plugin. You can turn on/off an option by adding/removing the corresponding flag
 some meaningful syntax in the source file. This option is enabled by default.
 
 
+
 ## Developer's Guide
 
-To set up your development environment, perform the following steps:
+### Build from source
 1. Install the newest Node.js version for your operating system (OS); see the [Node.Js homepage](https://nodejs.org/en/).
 2. Install a Git client for your OS; see the [Git homepage](https://git-scm.com/downloads).
-3. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.)
-   See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
-4. Enable the corepack utility by running `corepack enable`, as superuser.
-   See the [Corepack homepage](https://github.com/nodejs/corepack).
-5. Run `npm install -g @vscode/vsce`.
-
-To start working on this project, perform the following steps:
-1. Create a working directory for the project and change into it.
-2. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
-   (adapt the URL to your specific fork and branch). 
-3. Change into the cloned directory `cd checker-framework-vscode`.
-4. Run `npm add -g @vscode/vsce`.
-
-To compile the downloader locally: 
+3. Create a working directory for the project and go into it.
+4. Clone the project in the new directory `git clone https://github.com/eisopux/checker-framework-vscode.git`
+5. Change into the cloned directory `cd checker-framework-vscode`.
+6. Install dependencies using `npm install`, as superuser. (Use `Open command prompt (CMD) (admin)` on Windows.) See the [npm Docs](https://docs.npmjs.com/cli/v8/commands/npm-install).
+7.  Run `npm install -g @vscode/vsce`, as superuser.
+8. Run `npm add -g @vscode/vsce`.
+9. In the `checker-framework-vscode` directory, run `vsce package` to generate a `.vsix` file 
+10. Run `code --install-extension checker-framework-language-server-0.2.0.vsix` 
```

Comment:
Maybe abstract out the version number and say to substitute the current version when doing this.

Reply:
@wmdietl I think this might complicate the instruction. For me, I like to simply copy the script and run it in my terminal. Maybe we can update it for every release or we remove the version number from the file name?

Reply:
Sorry, it seems like the second option is not good. Maybe we can update it for every release?